aea/configurations/base.py

```diff
@@ -1203,26 +1260,28 @@ def default_ledger(self, ledger_id: str):
     @property
     def json(self) -> Dict:
         """Return the JSON representation."""
-        return {
-            "agent_name": self.agent_name,
-            "author": self.author,
-            "version": self.version,
-            "license": self.license,
-            "aea_version": self.aea_version,
-            "fingerprint": self.fingerprint,
-            "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
-            "registry_path": self.registry_path,
-            "description": self.description,
-            "private_key_paths": self.private_key_paths_dict,
-            "ledger_apis": self.ledger_apis_dict,
-            "logging_config": self.logging_config,
-            "default_ledger": self.default_ledger,
-            "default_connection": self.default_connection,
-            "connections": sorted(map(str, self.connections)),
-            "protocols": sorted(map(str, self.protocols)),
-            "skills": sorted(map(str, self.skills)),
-            "contracts": sorted(map(str, self.contracts)),
-        }
+        return OrderedDict(
+            {
+                "agent_name": self.agent_name,
+                "author": self.author,
+                "version": self.version,
+                "description": self.description,
+                "license": self.license,
+                "aea_version": self.aea_version,
+                "fingerprint": self.fingerprint,
+                "fingerprint_ignore_patterns": self.fingerprint_ignore_patterns,
```

Comment:
In spirit of above, would have: "connections", "contracts", "protocols", "skills", "default_connection", "default_ledger", "ledger_apis", "logging_config", "private_key_paths", "registry_path"